src/test/java/com/klarna/hiverunner/examples/HelloHiveRunnerParamaterized.java

```diff
@@ -0,0 +1,70 @@
+/**
+ * Copyright (C) 2013-2020 Klarna AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.klarna.hiverunner.examples;
+
+import static org.junit.Assert.assertArrayEquals;
+import static org.junit.Assert.assertEquals;
+
+import java.util.List;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import com.klarna.hiverunner.HiveRunnerExtension;
+import com.klarna.hiverunner.HiveShell;
+import com.klarna.hiverunner.annotations.HiveSQL;
+
+/**
+ * A basic Hive Runner example showing how to use JUnit5's ParameterizedTest.
+ */
+@ExtendWith(HiveRunnerExtension.class)
+public class HelloHiveRunnerParamaterized {
+
+    @HiveSQL(files = {})
+    private HiveShell shell;
+
+    @BeforeEach
+    public void setupSourceDatabase() {
+        shell.executeStatement("CREATE DATABASE source_db");
+    }
+
+    @ParameterizedTest
+    @ValueSource(strings = { "sequencefile", "ORC", "PARQUET" })
```

Comment:
Why the difference in case? Is there any convention on how to insert the file format in Hive?

Reply:
I don't think Hive cares, to be consistent let me just make this all upper case as that's usually how I've seen it done in DDL statements.

Reply:
+1, I've seen the same.